packages/ui/pages/taskdetail.tsx

```diff
@@ -0,0 +1,347 @@
+              fontWeight: 'bold',
+              marginRight: '10px',
+            }}
+          >
+            Understand knock-in mouse model
+          </h1>
+          <AntButton onClick={() => {}} icon={<MoreOutlined />} />
+        </div>
+        <p style={{ color: 'lightgray' }}>
+          This task is to gain an understanding of what knock-in mouse
+          model is, the high level process to design and build and the
+          cost/time it takes to develop a model. Please work with an
+          expert to determine if this is the right model for you.
+        </p>
+      </section>
+      <section
+        style={{
+          padding: '20px',
+          display: 'flex',
+          justifyContent: 'space-between',
+        }}
+      >
+        <div
+          style={{
+            width: '60%',
+
+            backgroundColor: '#ffffff',
+            padding: '2rem',
+          }}
+        >
+          <div
+            style={{
+              display: 'flex',
+              justifyContent: 'space-between',
+            }}
+          >
+            <div style={{ display: 'flex', flexDirection: 'column' }}>
+              <span>Status</span>
+              <span
+                style={{
+                  backgroundColor: 'orange',
+                  color: '#fff',
+                  padding: '1px 2px',
+                }}
+              >
+                In Progress
+              </span>
+            </div>
+            <div
+              style={{
+                display: 'flex',
+                flexDirection: 'column',
+                position: 'relative',
+              }}
+            >
+              <span>Owner</span>
+              <span
+                style={{
+                  display: 'flex',
+                  padding: '0 4px',
+                  border: '1px solid lightblue',
+                }}
+              >
+                <LetterPic letter="R" size="sm" />{' '}
+                <span style={{ marginLeft: '4px' }}>Ramya</span>
+                <span>
+                  {isUserDropDownOpen ? (
+                    <svg
+                      onClick={() =>
+                        setUserDropDown(!isUserDropDownOpen)
+                      }
+                      style={{ width: '25px' }}
+                      xmlns="http://www.w3.org/2000/svg"
+                      viewBox="0 0 20 20"
+                      fill="currentColor"
+                    >
+                      <path
+                        fillRule="evenodd"
+                        d="M5.293 7.293a1 1 0 011.414 0L10 10.586l3.293-3.293a1 1 0 111.414 1.414l-4 4a1 1 0 01-1.414 0l-4-4a1 1 0 010-1.414z"
+                        clipRule="evenodd"
+                      />
+                    </svg>
+                  ) : (
+                    <svg
+                      onClick={() =>
+                        setUserDropDown(!isUserDropDownOpen)
+                      }
+                      style={{ width: '25px' }}
```

Comment:
@pipesort123TP can we use the Icon component and use onClick on it? instead of inlining the svg here?